map/src/drawer/components/favorite/FavoriteItem.jsx

```diff
@@ -0,0 +1,64 @@
+import {ListItemText, MenuItem, Tooltip, Typography} from "@mui/material";
+import React, {useContext} from "react";
+import AppContext from "../../../context/AppContext";
+import {makeStyles} from "@material-ui/core/styles";
+
+
+const useStyles = makeStyles({
+    icon: {
+        "& .icon": {
+            top: '14px',
+            left: '30px',
+        },
+        "& .background": {
+            left: '0px',
+            top: '2px',
+            filter: "drop-shadow(0 0 0 gray)"
+        }
+    },
+    item: {
+        '& li': {
+            minHeight: '50px !important',
+            maxHeight: '50px !important',
+        }
+    },
+    text: {
+        '& p': {
```

Comment:
It looks strange cause we didn't use any custom style for other elements in drawer. Needs to be investigated.

Reply:
We don't have do it other way. We use makeStyles - personal styles for components. This can be changed separately if needed.